src/main.rs

```diff
@@ -597,7 +597,7 @@ fn maybe_write_config_file(opt: CliOptions, password: Option<String>) -> PathBuf
         println!("Using {} as connection string", c);
         c
     } else {
-        println!("Please enter connection string:");
+        println!("Please enter connection string (.e.g. https://10.112.210.101:18091,10.112.210.101:18091)");
```

Comment:
Why wouldn't we use "couchbases://cb.<yourdb>.sdk.cloud.couchbase.com/" ?  I do recognize that cbsh leverages mostly the REST interface, but generally a connstr should be couchbase:// or whatever we come up for that other thing.

Reply:
1) The same IP address is twice.  There is no advantage to that - in fact there is a disadvantage.

2) the port number cannot be repeated. It may have been accepted by some SDKs, but it should not have been.

3) What Matt said.

Reply:
Discussed most recent change with Matt and agreed looks good.